### DISCUSSION TITLE
Annoucing Dear ImGui Bundle including ImPlot and many other goodies; with support for C++, Python, and notebook

### DISCUSSION THREAD

**Discussion by pthom** [epezent/implot#441] [Show and tell]
I'm happy to announce the first public release of Dear ImGui Bundle. Dear ImGui Bundle is a bundle for Dear ImGui, including many powerful libraries from its ecosystem. It enables to easily create ImGui applications in C++ *and* Python, under Windows, macOS, Linux, emscripten. It is aimed at application developers, researchers, and beginner developers who want to quickly get started.

ImPlot is a central part of it; and it is a joy to use ImPlot with the same ease for emscripten, C++, python, and even notebooks: 

<img src="https://traineq.org/imgui_bundle_doc/demo_bundle8.gif" width=400>

<img width="300" alt="image" src="https://user-images.githubusercontent.com/7694091/214649637-d0c5f599-e786-4242-8cfd-d0b54e5956cb.png">

* [Dear ImGui Bundle repo](https://github.com/pthom/imgui_bundle/)
* [Fully interactive online manual](https://traineq.org/ImGuiBundle/emscripten/bin/demo_imgui_bundle.html)

@epezent : my apologies if I inform you a bit late about this library. I've been working for a long time discreetly on it, and I wanted to wait until it was a bit battle tested before communicating about it. I wrote a specific [annoucement related to ImPlot integration](https://github.com/pthom/imgui_bundle/discussions/64), which describes more in detail how the integration works. I informed @ocornut [yesterday](https://twitter.com/ocornut/status/1617963615503659008), and I plan to communicate about it in the [ImGui gallery](https://github.com/ocornut/imgui/issues/5886) in the next hours.

**Comment by brenocq** [comment]
So. good.

**Reply by brenocq** [reply]
<img width="1279" height="102" alt="image" src="https://github.com/user-attachments/assets/5c8442ce-35b1-41e6-8138-29dff8a71ab3" />
Omg!!!!

**Reply by pthom** [reply]
Thank you !

**Reply by brenocq** [reply]
<img width="300" src="https://github.com/user-attachments/assets/eb9e909e-4570-4a38-9432-5ff9fc9f8a32"/>

Congrats Pascal!! 🚀 🚀
